rackcity/api/serializers/asset_serializers.py

```diff
@@ -193,11 +221,32 @@ def normalize_bulk_asset_data(bulk_asset_data):
     return bulk_asset_data
 
 
-def serialize_power_connections(asset):
-    try:
-        ports = PowerPort.objects.filter(asset=asset.id)
-    except ObjectDoesNotExist:
-        return
+def serialize_mac_addresses(network_port_model, asset):
+    ports = network_port_model.objects.filter(asset=asset.id)
```

Comment:
Removed the try/except around this because `.filter` does not throw doesnotexist exceptions--it will just return an empty queryset if no existing rows match the filter